src/test/java/com/synopsys/integration/jenkins/scan/service/bridge/BridgeDownloadParameterServiceTest.java

```diff
@@ -88,6 +88,53 @@ void getBridgeDownloadParamsTest() {
         assertEquals("/path/to/bridge", result.getBridgeInstallationPath());
     }
 
+    @Test
+    void getBridgeDownloadParamsTestWithAirgapEnabledAndVersion() {
+        Map<String, Object> scanParams = new HashMap<>();
+        scanParams.put(ApplicationConstants.BRIDGE_DOWNLOAD_VERSION, "3.0.0");
+        scanParams.put(ApplicationConstants.BRIDGE_INSTALLATION_PATH, "/path/to/bridge");
+        scanParams.put(ApplicationConstants.BRIDGE_NETWORK_AIRGAP_KEY, true);
+
+        BridgeDownloadParameters
+            bridgeDownloadParameters = new BridgeDownloadParameters(workspace, listenerMock);
+
+        BridgeDownloadParameters result = bridgeDownloadParametersService.getBridgeDownloadParams(scanParams, bridgeDownloadParameters);
+
+        assertFalse(result.getBridgeDownloadUrl().contains(".zip"));
+        assertEquals("/path/to/bridge", result.getBridgeInstallationPath());
+    }
+
+    @Test
+    void getBridgeDownloadParamsTestForAirgap() {
```

Comment:
The method name should be  `getBridgeDownloadParamsForAirgapTest()` so that all test method names are consistent.

Reply:
Done